edb/server/main.py

```diff
@@ -491,6 +511,10 @@ def _get_runstate_dir_default() -> str:
         '--auto-shutdown', type=bool, default=False, is_flag=True,
         help='shutdown the server after the last management ' +
              'connection is closed'),
+    click.option(
+        '--max-protocol', type=str, callback=_protocol_version,
```

Comment:
Perhaps I missed something, but why do we need this flag?

Reply:
It was my suggestion initially.  We need this for the ability to test protocol downgrade in clients, but since we agreed that we'd be dropping support for older protocols in clients for now, this isn't strictly necessary now.